docs_api/h5ds.rst

```diff
@@ -0,0 +1,20 @@
+Module H5DS
+===========
+
+.. automodule:: h5py.h5ds
+    :members:
```

Comment:
The `:members:` option tries to find and document functions inside the module automatically - so Sphinx is complaining because some of the functions get documented twice.

I've resolved this by deleting the list below, and adding `:undoc-members:` here to include the functions which don't yet have a docstring.

Reply:
Oh, I see. Thanks😃